src/Consolonia.Gallery/View/ControlsListView.axaml.cs

```diff
@@ -9,6 +9,7 @@
 using Avalonia.Controls.ApplicationLifetimes;
```

Comment:
Closing settings dialog or cancelling menu popup leaves the focus somewhere out of text.